h/views/api.py

```diff
@@ -79,7 +79,7 @@ def add_api_view(config, view, link_name=None, description=None, **settings):
         request_method = (request_method,)
     if len(request_method) == 0:
         request_method = ('DELETE', 'GET', 'HEAD', 'PATCH', 'POST', 'PUT',)
-    settings['request_method'] = request_method + ('OPTIONS',)
+    settings['request_method'] = request_method
```

Comment:
This whole block starting from 77 to this new line can be removed now. It was added in b70267229817e376a0cfb11f1c5118f0c6f4058c to be able to add the `OPTIONS` method. But we had to support all the different ways that the `request_method` could have been set on the view itself, which is why it's a whole block of code.

Also note that the line before where we add all possible request methods was only there because we've added `OPTIONS`, by default in pyramid a view responds to all methods, but if we add `OPTIONS` to an empty list, then we take away support for any other request method.

But now we don't need to add `OPTIONS` to a potential list of methods anymore, so we don't need to change the `request_method` setting at all.